src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -2619,7 +2619,13 @@ def __init__(self, cmd, client, linux=False):
     @staticmethod
     def remove_delimiters(runtime):
         import re
-        runtime = re.split('[| :]', runtime)  # delimiters allowed: '|', ' ', ':'
+        # delimiters allowed: '|', ':', ' '
```

Comment:
When a user use hybrid delimiter, what will happen? For example, java|1.8:Java SE 8

Reply:
Should we cancel delimiter ' '? It is confusing. "Java SE" should not be split. I don't think it is a good delimiter because it is a valid character in runtime name.

Reply:
Hybrid delimiter won't work. I've updated to remove space as a delimiter, and docs show use '|' or ':'